serverless.yml

```diff
@@ -91,3 +91,20 @@ functions:
         #GITHUB_APP_ID: ${ssm:/${self:provider.stage}/${self:service}/github/app/id}
     events:
         - sns: ${self:custom.topic.diff}
+        
+# TODO Upload file to created bucket
+resources:
+  Resources:
+    UploadBucket:
+      Type: AWS::S3::Bucket
+      Properties:
+        BucketName: "${self:service}-${self:provider.stage}-statichomepagebucket"
```

Comment:
will this replace the existing bucket for this service/stage if it is run twice?

Reply:
That's up to CloudFormation - this plugs into AWS's existing automation. I believe so - I ran multiple times and it only created one